contracts/superfluid/v2/TradableStream.sol

```diff
@@ -23,6 +23,8 @@ struct TradableStreamMetadata {
 uint256 constant MAX_DURATION_SECONDS = 60 * 60 * 24 * 30;
```

Comment:
We should not hard code 30 days as the max duration. I am sure we want to have duration that is much longer than 30 days in the future.